doc/userguide/rules/datasets.rst

```diff
@@ -169,6 +169,50 @@ type
 data
   Data to remove in serialized form (base64 for string, hex notation for md5/sha256)
 
+dataset-remove
+~~~~~~~~~~~~~~
+
+Unix Socket command to remove all data from a set. On success, the removal becomes
+active instantly.
+
+Syntax::
+
+    dataset-clear <set name> <set type>
+
+set name
+  Name of an already defined dataset
+type
+  Data type: string, md5, sha256
+
+dataset-exist
```

Comment:
I'd be in favor of a different name. Something like `dataset-match` or `dataset-lookup` or something similar.

Reply:
dataset-lookup will match the hash table code?